### ISSUE TITLE
Adding Object Tracking and Re-Identification Using YOLOV9 and Strong SORT and OSNET

### ISSUE THREAD

**Issue by TheNobody-12** [WongKinYiu/yolov9#299]
# Description
This Release contains a highly configurable two-stage tracker that adjusts to different deployment scenarios. The detections generated by [YOLOv9](https://github.com/WongKinYiu/yolov9), a family of object detection architectures and models pre-trained on the COCO dataset, are passed to [StrongSORT](https://github.com/dyhBUPT/StrongSORT) which combines motion and appearance information based on [OSNet](https://github.com/KaiyangZhou/deep-person-reid) to track the objects. It can track any object that your Yolov9 model was trained to detect.

# Example of Tracking Algorithm
![003-gif](https://github.com/TheNobody-12/MOT_WITH_YOLOV9_STRONG_SORT/assets/75840118/7ae8f898-4db3-416d-b579-4ddffc339e0d)

# Key features:
🎯 Accurate object detection powered by YOLOv9
🔄 Seamless object tracking using StrongSORT and OSNet
🔧 Highly configurable to adapt to different deployment scenarios
# Repository Link
[The YOLOv9_StrongSORT_OSNET](https://github.com/TheNobody-12/MOT_WITH_YOLOV9_STRONG_SORT)

**Comment by TheNobody-12** [comment]
@WongKinYiu Do check this out

**Comment by JFMeyer2k** [comment]
Do you have comparisons of performance to other trackers like (older) trackers like ByteTrack or DeepSORT?

**Comment by TheNobody-12** [comment]
Yes, the STRONG sort is the advancement to the Deep SORT framework. 
https://github.com/dyhBUPT/StrongSORT
here is the original code repository of the StrongSORT which performs better than ByteTrack and  DeepSort. 

# Results:
![image](https://github.com/WongKinYiu/yolov9/assets/75840118/a6797e94-4533-4293-8075-7897c2dbd972)

**Comment by WongKinYiu** [comment]
Added to readme.
